Strange symbols when upgrading to v14.0.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

Short story: Got a new computer, installed oh-my-posh with my custom theme started to get strange symbols on-enter (see screenshot below):
<img width="242" alt="Screenshot 2023-01-31 at 16 59 24" src="https://user-images.githubusercontent.com/21131005/215813457-44b048d5-11ef-45a7-bb15-37287e0be6df.png">

I'm using Warp + custom theme, see attached theme.

---

I temporary resolved it by manually downgrading. 

I copied over the contents from my old machine:
``` sh
cd /usr/local/Cellar/oh-my-posh/13.3.0/
```
and pasted the content over to my new computer:
```sh
cd /opt/homebrew/Cellar/oh-my-posh/14.0.0
```
which resolves it, see screenshot (prompt censured):
<img width="317" alt="Screenshot 2023-01-31 at 17 07 47" src="https://user-images.githubusercontent.com/21131005/215815535-fd459573-42ed-4a5e-aed6-0bd76416fb25.png">

which is not ideal, but it works.. 

---

Question: Any clues on why I got those strange symbols?

### Theme

``` json
{
  "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
  "version": 2,
  "console_title_template": "{{ .UserName }}@{{ .HostName }}",
  "terminal_background": "p:background",
  "blocks": [
    {
      "type": "prompt",
      "alignment": "left",
      "segments": [
        {
          "type": "session",
          "style": "plain",
          "foreground": "p:yellow",
          "template": "{{ .UserName }}🍺 "
        },
        {
          "type": "path",
          "style": "plain",
          "foreground": "p:blue",
          "template": "{{ .Path }} ",
          "properties": {
            "style": "full"
          }
        },
        {
          "background": "p:background-color",
          "foreground": "p:cloud-text-gcp",
          "style": "powerline",
          "template": "<p:cyan>{{ .Project }}</> ",
          "type": "gcp"
        },
        {
          "foreground": "#5fb157",
          "properties": {
            "display_mode": "files",
            "fetch_package_manager": true,
            "fetch_version": true,
            "npm_icon": "<#FE4A49>npm</>",
            "yarn_icon":"<#44FFD2>yarn</>"
          },
          "style": "plain",
          "template": "{{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}",
          "type": "node"
        },
        {
          "type": "git",
          "style": "plain",
          "template": "<p:yellow>git</>:({{ if or (.Working.Changed) (.Staging.Changed) (gt .StashCount 0) }}<p:magenta>{{ .HEAD }}</>{{ else }}<p:green>{{ .HEAD }}</>{{ end }}{{ if (gt .Ahead 0)}}<p:cyan>{{ .BranchStatus }}</>{{ end }}{{ if (gt .Behind 0)}}<p:cyan>{{ .BranchStatus }}</>{{ end }}{{ if .Staging.Changed }} <p:green>{{ .Staging.String }}</>{{ end }}{{ if .Working.Changed }} <p:red>{{ .Working.String }}</>{{ end }})",
          "properties": {
            "branch_icon": "",
            "fetch_stash_count": true,
            "fetch_status": true,
            "fetch_worktree_count": true
          }
        }
      ],
      "newline": true
    }
  ],
  "palette": {
    "background": "#282c34",
    "black": "#282c34",
    "blue": "#61afef",
    "cyan": "#56b6c2",
    "foreground": "#dcdfe4",
    "green": "#98c379",
    "magenta": "#c678dd",
    "red": "#e06c75",
    "white": "#dcdfe4",
    "yellow": "#e5c07b"
  }
}
```

### What OS are you seeing the problem on?

macOS

### Which shell are you using?

zsh

### Log output

```Shell
oh-my-posh debug

Version: 14.0.0

Segments:

ConsoleTitle(true) -   0 ms - <censored>@GOT5CG7041DD2.internal.s1.sto.swe.m.co
session(true)      -   0 ms - <censored>🍺
path(true)         -   0 ms - ~
gcp(true)          -   0 ms - <censored>
node(false)        -   0 ms -
git(false)         -   0 ms -

Run duration: 5.533417ms

Cache path: /Users/<censored>

Config path: /Users/<censored>/.<censored>design.omp.json

Logs:

[DEBUG] 17:13:39.753 shell.go:Getenv:342 → NO DATA
[TRACE] 17:13:39.753 shell.go:Getenv(XDG_CACHE_HOME) - 873.75µs
[TRACE] 17:13:39.753 shell_unix.go:CachePath() - 893.709µs
[DEBUG] 17:13:39.756 shell.go:Getenv:342 → /Users/<censored>/.<censored>design.omp.json
[TRACE] 17:13:39.756 shell.go:Getenv(POSH_THEME) - 3.666µs
[TRACE] 17:13:39.756 shell.go:resolveConfigPath() - 6.875µs
[TRACE] 17:13:39.756 shell.go:Init() - 3.476ms
[TRACE] 17:13:39.756 shell.go:Flags() - 41ns
[TRACE] 17:13:39.756 config.go:loadConfig() - 255.417µs
[TRACE] 17:13:39.756 shell.go:Flags() - 42ns
[DEBUG] 17:13:39.756 shell.go:Getenv:342 → NO DATA
[TRACE] 17:13:39.756 shell.go:Getenv(OMP_CACHE_DISABLED) - 1.792µs
[TRACE] 17:13:39.756 shell_unix.go:Root() - 542ns
[DEBUG] 17:13:39.756 shell.go:Shell:587 → process name: zsh
[TRACE] 17:13:39.756 shell.go:Shell() - 268.584µs
[TRACE] 17:13:39.756 shell.go:ErrorCode() - 83ns
[DEBUG] 17:13:39.756 shell.go:Getenv:342 → NO DATA
[TRACE] 17:13:39.756 shell.go:Getenv(WSL_DISTRO_NAME) - 2.125µs
[TRACE] 17:13:39.756 shell_unix.go:IsWsl() - 4.125µs
[TRACE] 17:13:39.756 shell.go:GOOS() - 41ns
[DEBUG] 17:13:39.756 shell.go:Pwd:370 → NO DATA
[TRACE] 17:13:39.756 shell.go:Pwd() - 20.209µs
[TRACE] 17:13:39.756 shell.go:GOOS() - 83ns
[TRACE] 17:13:39.756 shell.go:GOOS() - 42ns
[TRACE] 17:13:39.756 shell.go:GOOS() - 0s
[TRACE] 17:13:39.756 shell.go:GOOS() - 42ns
[TRACE] 17:13:39.756 shell.go:GOOS() - 0s
[TRACE] 17:13:39.756 shell.go:GOOS() - 0s
[DEBUG] 17:13:39.756 shell.go:User:491 → <censored>
[TRACE] 17:13:39.756 shell.go:User() - 2.917µs
[DEBUG] 17:13:39.757 shell.go:Host:503 → GOT5CG7041DD2.internal.s1.sto.swe.m.co
[TRACE] 17:13:39.757 shell.go:Host() - 133.709µs
[TRACE] 17:13:39.757 shell.go:GOOS() - 42ns
[TRACE] 17:13:39.757 shell.go:TemplateCache() - 472.125µs
[TRACE] 17:13:39.757 shell.go:Flags() - 42ns
[DEBUG] 17:13:39.757 shell.go:Pwd:350 → /Users/<censored>
[TRACE] 17:13:39.757 shell.go:Pwd() - 2.042µs
[TRACE] 17:13:39.757 shell.go:GOOS() - 41ns
[DEBUG] 17:13:39.757 shell.go:Getenv:342 → NO DATA
[TRACE] 17:13:39.757 shell.go:Getenv(SSH_CONNECTION) - 3.833µs
[DEBUG] 17:13:39.757 shell.go:Getenv:342 → NO DATA
[TRACE] 17:13:39.757 shell.go:Getenv(SSH_CLIENT) - 1.5µs
[TRACE] 17:13:39.757 shell.go:TemplateCache() - 83ns
[TRACE] 17:13:39.757 shell.go:TemplateCache() - 84ns
[TRACE] 17:13:39.757 shell.go:Shell() - 83ns
[DEBUG] 17:13:39.757 shell.go:Pwd:350 → /Users/<censored>
[TRACE] 17:13:39.757 shell.go:Pwd() - 2.375µs
[TRACE] 17:13:39.757 shell.go:GOOS() - 41ns
[DEBUG] 17:13:39.757 shell.go:Pwd:350 → /Users/<censored>
[TRACE] 17:13:39.757 shell.go:Pwd() - 1.625µs
[TRACE] 17:13:39.757 shell.go:Shell() - 41ns
[TRACE] 17:13:39.757 shell.go:Shell() - 42ns
[TRACE] 17:13:39.757 shell.go:GOOS() - 41ns
[TRACE] 17:13:39.757 shell.go:PathSeparator() - 42ns
[TRACE] 17:13:39.757 shell.go:GOOS() - 41ns
[TRACE] 17:13:39.757 shell.go:GOOS() - 0s
[TRACE] 17:13:39.757 shell.go:GOOS() - 42ns
[TRACE] 17:13:39.757 shell.go:GOOS() - 0s
[TRACE] 17:13:39.757 shell.go:PathSeparator() - 42ns
[TRACE] 17:13:39.757 shell.go:PathSeparator() - 42ns
[TRACE] 17:13:39.757 shell.go:GOOS() - 0s
[TRACE] 17:13:39.757 shell.go:GOOS() - 41ns
[TRACE] 17:13:39.757 shell.go:PathSeparator() - 42ns
[TRACE] 17:13:39.757 shell.go:GOOS() - 0s
[TRACE] 17:13:39.757 shell.go:GOOS() - 42ns
[TRACE] 17:13:39.757 shell.go:PathSeparator() - 41ns
[TRACE] 17:13:39.757 shell.go:GOOS() - 0s
[TRACE] 17:13:39.757 shell.go:GOOS() - 0s
[TRACE] 17:13:39.757 shell.go:PathSeparator() - 42ns
[DEBUG] 17:13:39.757 shell.go:Pwd:350 → /Users/<censored>
[TRACE] 17:13:39.757 shell.go:Pwd() - 4.083µs
[DEBUG] 17:13:39.757 shell.go:Getenv:342 → NO DATA
[TRACE] 17:13:39.757 shell.go:Getenv(WSL_DISTRO_NAME) - 1.5µs
[TRACE] 17:13:39.757 shell_unix.go:IsWsl() - 3.25µs
[TRACE] 17:13:39.757 shell.go:StackCount() - 41ns
[TRACE] 17:13:39.757 shell_unix.go:DirIsWritable(/Users/<censored>) - 11.917µs
[TRACE] 17:13:39.757 shell.go:TemplateCache() - 83ns
[TRACE] 17:13:39.757 shell.go:TemplateCache() - 84ns
[TRACE] 17:13:39.757 shell.go:Shell() - 42ns
[DEBUG] 17:13:39.757 shell.go:Pwd:350 → /Users/<censored>
[TRACE] 17:13:39.757 shell.go:Pwd() - 1.916µs
[TRACE] 17:13:39.757 shell.go:GOOS() - 41ns
[DEBUG] 17:13:39.757 shell.go:Getenv:342 → NO DATA
[TRACE] 17:13:39.757 shell.go:Getenv(CLOUDSDK_CONFIG) - 1.625µs
[TRACE] 17:13:39.757 shell.go:GOOS() - 42ns
[DEBUG] 17:13:39.757 shell.go:FileContent:459 → default
[TRACE] 17:13:39.757 shell.go:FileContent(/Users/<censored>/.config/gcloud/active_config) - 227.542µs
[DEBUG] 17:13:39.757 shell.go:FileContent:459 ↓
    [core]
    account = <censored>rey.ornerstedt@m.co
    project = <censored>


[TRACE] 17:13:39.757 shell.go:FileContent(/Users/<censored>/.config/gcloud/configurations/config_default) - 24.917µs
[TRACE] 17:13:39.757 shell.go:TemplateCache() - 83ns
[TRACE] 17:13:39.757 shell.go:TemplateCache() - 83ns
[TRACE] 17:13:39.757 shell.go:Shell() - 42ns
[DEBUG] 17:13:39.758 shell.go:Pwd:350 → /Users/<censored>
[TRACE] 17:13:39.758 shell.go:Pwd() - 1.959µs
[TRACE] 17:13:39.758 shell.go:GOOS() - 42ns
[DEBUG] 17:13:39.758 shell.go:Pwd:350 → /Users/<censored>
[TRACE] 17:13:39.758 shell.go:Pwd() - 1.75µs
[DEBUG] 17:13:39.758 shell.go:Pwd:350 → /Users/<censored>
[TRACE] 17:13:39.758 shell.go:Pwd() - 1.666µs
[TRACE] 17:13:39.758 shell.go:GOOS() - 41ns
[DEBUG] 17:13:39.758 shell.go:Getenv:342 → NO DATA
[TRACE] 17:13:39.758 shell.go:Getenv(WSL_DISTRO_NAME) - 1.417µs
[TRACE] 17:13:39.758 shell_unix.go:IsWsl() - 3.125µs
[TRACE] 17:13:39.758 shell_unix.go:IsWsl2() - 7.625µs
[TRACE] 17:13:39.758 shell.go:GOOS() - 41ns
[DEBUG] 17:13:39.758 shell.go:CommandPath:542 → /usr/bin/git
[TRACE] 17:13:39.758 shell.go:CommandPath(git) - 107.75µs
[TRACE] 17:13:39.758 shell.go:HasCommand(git) - 110.958µs
[DEBUG] 17:13:39.758 shell.go:Pwd:350 → /Users/<censored>
[TRACE] 17:13:39.758 shell.go:Pwd() - 1.708µs
[ERROR] 17:13:39.758 shell.go:HasParentFilePath:675 → stat /.git: no such file or directory
[TRACE] 17:13:39.758 shell.go:HasParentFilePath(.git) - 30.292µs
[DEBUG] 17:13:39.758 shell.go:Getenv:342 → NO DATA
[TRACE] 17:13:39.758 shell.go:Getenv(XDG_CACHE_HOME) - 1.584µs
[TRACE] 17:13:39.758 shell_unix.go:CachePath() - 6.625µs
[TRACE] 17:13:39.758 shell.go:Flags() - 42ns
```

--- a/src/shell/scripts/omp.bash
+++ b/src/shell/scripts/omp.bash
@@ -12,7 +12,7 @@ PS2="$(::OMP:: print secondary --config="$POSH_THEME" --shell=bash --shell-versi
 function _set_posh_cursor_position() {
     # not supported in Midnight Commander
     # see https://github.com/JanDeDobbeleer/oh-my-posh/issues/3415
-    if [[ -v MC_SID ]];then
+    if [[ -v MC_SID ]] || [[ $TERM_PROGRAM == "WarpTerminal" ]]; then
         return
     fi
     local pos

--- a/src/shell/scripts/omp.zsh
+++ b/src/shell/scripts/omp.zsh
@@ -10,7 +10,7 @@ PS2="$(::OMP:: print secondary --config="$POSH_THEME" --shell=zsh)"
 function _set_posh_cursor_position() {
   # not supported in Midnight Commander
   # see https://github.com/JanDeDobbeleer/oh-my-posh/issues/3415
-  if [[ -v MC_SID ]];then
+  if [[ -v MC_SID ]] || [[ $TERM_PROGRAM == "WarpTerminal" ]]; then
       return
   fi
   local pos

